Marlin/language_es.h

```diff
@@ -167,5 +167,47 @@
 #define MSG_DELTA_CALIBRATE_Y               "Calibrar Y"
 #define MSG_DELTA_CALIBRATE_Z               "Calibrar Z"
 #define MSG_DELTA_CALIBRATE_CENTER          "Calibrar Centro"
-
+#define MSG_INFO_MENU                       "Inf. Impresora"
+#define MSG_INFO_PRINTER_MENU               "Inf. Impresora"
+#define MSG_INFO_STATS_MENU                 "Estadisticas Imp."
+#define MSG_INFO_BOARD_MENU                 "Inf. Controlador"
+#define MSG_INFO_THERMISTOR_MENU            "Termistores"
+#define MSG_INFO_EXTRUDERS                  "Extrusores"
+#define MSG_INFO_BAUDRATE                   "Baudios"
+#define MSG_INFO_PROTOCOL                   "Protocolo"
+#define MSG_INFO_PRINT_COUNT                "Conteo de impresion"
+#define MSG_INFO_COMPLETED_PRINTS           "Completadas"
+#define MSG_INFO_PRINT_TIME                 "Tiempo total de imp."
+#define MSG_INFO_PRINT_LONGEST              "Impresion mas larga"
+#define MSG_INFO_PRINT_FILAMENT             "Total de Extrusion"
+#define MSG_INFO_PRINT_COUNT                "Impresiones"
+#define MSG_INFO_COMPLETED_PRINTS           "Completadas"
+#define MSG_INFO_PRINT_TIME                 "Total"
+#define MSG_INFO_PRINT_LONGEST              "Mas larga"
+#define MSG_INFO_PRINT_FILAMENT             "Extrusion"
+#define MSG_INFO_MIN_TEMP                   "Temperatura minima"
+#define MSG_INFO_MAX_TEMP                   "Temperatura maxima"
+#define MSG_INFO_PSU                        "Fuente de poder"
+#define MSG_FILAMENT_CHANGE_HEADER          "Cambiar Filamento"
+#define MSG_FILAMENT_CHANGE_OPTION_HEADER   "Opciones de cambio:"
+#define MSG_FILAMENT_CHANGE_OPTION_EXTRUDE  "Extruir mas"
+#define MSG_FILAMENT_CHANGE_OPTION_RESUME   "Resumir imp."
+#define MSG_FILAMENT_CHANGE_INIT_1          "Esperando para empesar"
```

Comment:
Characters past 20 will never be seen, so try to make this string shorter. I find that a "ruler" helps:
```
12345678901234567|90
Esperando para empesar
```